### ISSUE TITLE
Add the new transformer architecture to the vanilla one

### ISSUE THREAD

**Issue by shanika04** [Just1truc/share-qa#3]
*  Freeze the MLP and old attention mechanism
* Add loss (NLL / MSE) between old-attention logits and new ones
* Train using the loss above + same dataset as the previous one
